components/Awards.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const Awards = props => {
+  return (
+    <div>
+      <h3>
+        {props.data.title} - {props.data.date}
+      </h3>
+      <h4>{props.data.awarder}</h4>
+      <p>{props.data.summary}</p>
+      <style jsx>{`
```

Comment:
I haven't worked with nextjs in the past so I'm not sure about conventions, but maybe some alternatives to using a style tag to apply css would be some sort of css-in-js library or external css files to separate the UI style from the UI content

Reply:
these style tags are actually Next's built in css-in-js library!